src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,84 @@
+package homework_2.random_chars_table;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.InputMismatchException;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class RandomCharsTable {
+
+    private static void randomCharsTable(int lengthOfTable, int widthOfTable, String strategy) {
+
+        char[][] randomCharsTable = new char[lengthOfTable][widthOfTable];
+
+        int minRand = 'A';
+        int maxRand = 'Z';
+        for (int i = 0; i < lengthOfTable; i++) {
+            for (int j = 0; j < widthOfTable; j++) {
+                randomCharsTable[i][j] = (char) ((int) (Math.random() * (maxRand - minRand + 1) + minRand));
+                System.out.print(" | " + randomCharsTable[i][j] + " |");
+            }
+            System.out.println();
+        }
+
+        if ("even".equals(strategy.toLowerCase())) {
+            System.out.print("Even letters: ");
```

Comment:
```suggestion
            System.out.print("Even letters - ");
```

Reply:
fixed. thank you for your attentiveness;)